python/ray/serve/_private/benchmarks/serialization/common.py

```diff
@@ -0,0 +1,44 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+from ray.cloudpickle import cloudpickle
+
+#
+# NOTE: PLEASE READ CAREFULLY BEFORE CHANGING
```

Comment:
Maybe rename file to something like `payload_definitions` for clarity (looks like regular util file right now)